Lib/email/header.py

```diff
@@ -152,6 +158,23 @@ def decode_header(header):
     return collapsed
 
 
+
+def decode_header_to_string(header):
```

Comment:
I'm not seeing any tests of this function added in the PR.  Am I missing that (I'm possibly not navigating the iPadOS GitHub client correctly yet).